sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -562,10 +580,24 @@ func withSuperpowered(superpowered bool) oci.SpecOpts {
 	return oci.Compose(
 		oci.WithHostNamespace(runtimespec.PIDNamespace),
 		oci.WithParentCgroupDevices,
-		oci.WithPrivileged,
-		oci.WithNewPrivileges,
 		oci.WithSelinuxLabel("system_u:system_r:super_t:s0"),
 		oci.WithAllDevicesAllowed,
+	)
+}
+
+// withPrivilegedAccess adds container options to mount the root filesystem
+// as well as privileged threatment
```

Comment:
s/threatment/treatment/

Reply:
I removed the `withPrivilegedAccess`, since bootstrap containers shouldn't have all privileges just yet.